src/components/Nav/Nav.tsx

```diff
@@ -51,7 +51,6 @@ export class Nav extends React.Component<INavProps, INavState> {
           className={'ms-Nav-link' + (_isLinkSelected(link) ? ' is-selected' : '')}
           href={ link.url || 'javascript:' }
           onClick={ onLinkClick }
-          title={ link.title ? link.title : undefined }
```

Comment:
Those have different purposes. Title is for tooltips which may be desired (and possibly already in use) by others.